### PR TITLE
ACM-6703 ACM not ready warning

### PR DESCRIPTION
**Pull request by KevinFCormier**
https://issues.redhat.com/browse/ACM-6703

image

### CODE REVIEW

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -371,6 +371,8 @@
   "access.users": "Users",
   "access.usersInGroup": "Users in group",
   "access.usersInGroup.view": "View users in group",
+  "acm.plugin.not.ready": "The <bold>Red Hat Advanced Cluster Management for Kubernetes</bold> operator is installed, but the console plugin is not available. If operator installation has just completed, an alert should appear in a few moments prompting you to refresh the web console to access the latest version.",
```
Comment (dockerymick) [inline frontend/public/locales/en/translation.json:374]: Suggestion
```suggestion
  "acm.plugin.not.ready": "The <bold>Red Hat Advanced Cluster Management for Kubernetes</bold> operator is installed, but the console plugin is not available. If the operator installation completes, an alert appears in a few moments prompting you to refresh the web console to access the latest version.",
```

Comment (dockerymick) [reply frontend/public/locales/en/translation.json:374]: @KevinFCormier thanks for explaining your perspective and providing a suggestion. Your suggestion looks good to me!

**Review by robdolares — APPROVED** [review]
/lgtm

**Pull request merged**